mvpa2/clfs/gda.py

```diff
@@ -35,12 +35,12 @@
 from mvpa2.base.constraints import EnsureChoice
 from mvpa2.base.state import ConditionalAttribute
 #from mvpa2.measures.base import Sensitivity
-
+from sklearn.covariance import LedoitWolf, OAS
```

Comment:
since this code depends on sklearn presence it might be better to move it into a separate file (e.g. `shrinklda.py` or alike) which would help to import it conditionally on sklearn presence within the warehouse